client/src/components/LoginSession/tests/LoginSessionContainer-test.js

```diff
@@ -118,10 +114,9 @@ describe('LoginSessionContainer', () => {
       submitting: true
     });
 
-    httpResolve({
-      error: true,
-      success: false,
-      message: 'horrible failure'
+    httpReject({
+      response: {
```

Comment:
This mock an Error response. In a perfect world `response` would be returning a `Blob`, but Node 10 doesn't have that API (browsers do).

Manually creating a `text` method that returns a Promise is close enough. I've tested this in IE10 to make sure it works.